Contributors.md

```diff
@@ -0,0 +1,68 @@
+<p align="center">
+<a href="https://dscvit.com">
+	<img src="https://user-images.githubusercontent.com/30529572/72455010-fb38d400-37e7-11ea-9c1e-8cdeb5f5906e.png" />
+</a>
+</p>
+
+## Contributors
+
+<table>
+<tr align="center">
+
+
+<td>
+
+Insert person 1
+
+<p align="center">
+<img src = "https://dscvit.com/images/dsc-logo-square.svg" width="150" height="150" alt="Your Name Here (Insert Your Image Link In Src">
+</p>
+<p align="center">
+<a href = "https://github.com/person1"><img src = "http://www.iconninja.com/files/241/825/211/round-collaboration-social-github-code-circle-network-icon.svg" width="36" height = "36"/></a>
+<a href = "https://www.linkedin.com/in/person1">
+<img src = "http://www.iconninja.com/files/863/607/751/network-linkedin-social-connection-circular-circle-media-icon.svg" width="36" height="36"/>
+</a>
+</p>
+</td>
+
```

Comment:
Name placeholder or username placeholder not given and don't restrict the contributor from dsc only!

Reply:
Will add placeholder for the Name or Contributor. 

There are no restrictions as such. The contributor can be from anywhere, insert a link to their photo and change the name and social links.